documentation/src/main/markdown/runningdetect/autonomousscan.dita

```diff
@@ -0,0 +1,344 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+  PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
+<topic id="autonomous-scanning">
+    <title>Autonomous Scanning</title>
+    <shortdesc>Autonomous Scanning allows for <keyword keyref="company_name"/>
+        <keyword keyref="solution_name"/> to run scans with a minimum amount of user provided
+        parameters. Autonomous scans store configuration and related parameters in a .json file that
+        can be shared between users working with the same code. This reduces the user input required
```

Comment:
Can this file be shared? @gopannabd should we suggest users to share this file.

Reply:
No the file cannot be shared, the hashed name is derived on the basis of String value for local path, so it would not even work if it is shared. The file should only be used for rescanning purposes, no modification or sharing is expected from users.

Reply:
My view is not for the scan settings file to be (i.e. not recommended) shared. In subsequent releases, we will implement scan profiles that users can create and share. So, rather then share files, they can create a name for the profile and share with other team members or project teams.

Reply:
Well, I guess I will remove that comment from the doc and then just let the smart folks figure it out on their own. ; )